### DISCUSSION TITLE
GitHub Copilot 1.1.22.1530 and 1.1.24.1610 for JetBrains unsuable

### DISCUSSION THREAD

**Discussion by matquant14** [community/community#16876] [Copilot Conversations]
Just updated to GitHub Copilot 1.1.22.1530 on my JetBrains IDEs and it stopped working.  Seems to fail when trying to login at the start.  I've had to downgrade back to 1.1.20.1417

![2022-05-18 14 50 56](https://user-images.githubusercontent.com/43152300/169144349-3314c926-6698-4647-86e3-ca060983b08c.png)

**Comment by imxintian** [comment]
I am having a similar problem with JetBrains Golang.
When I updated to GitHub Copilot 1.1.22.1530, I could no longer use it. When I login to github via GitHub Copilot and pass the device-code authentication I get to the page https://github.com/login/device/success, but my JetBrains Golang stays on the waiting for github authentication... page.
<img width="660" alt="image" src="https://user-images.githubusercontent.com/20968904/169206113-9076375c-4bd9-45e4-b43c-8b2a6f72d291.png">


What should I do, please help.

**Reply by imxintian** [reply]
When I downgraded to version 1.1.20.147 Copilot, everything worked fine!

**Reply by jansorg** [reply]
@imxintian Thanks for reporting! Are you perhaps using a proxy?

**Reply by imxintian** [reply]
> @imxintian Thanks for reporting! Are you perhaps using a proxy?

In the same environment, I used a global vpn and did not use a proxy in Golang, only the downgrade was successful

**Reply by vincentfree** [reply]
I have the same issue behind a proxy. I've downgraded the plugin, now it works

**Reply by andrew222651** [reply]
PyCharm doesn't work properly with 1.1.22.1530 either, downgrading to 1.1.20.1417 still works.

**Reply by mrsunwei** [reply]
降级到 1.1.20.1417 还是无效。

**Comment by xixiaxixi** [comment]
Same issue. I got stuck waiting authentification same as @imxintian. After clicking cancel and try again, got result same as @matquant14 .

I tried using or not using proxy. Both can connect to github.io

**Comment by harrynull** [comment]
GitHub Copilot 1.1.22.1530 crashes my RubyMine instance after installing.

Log indicates 
```
Cyclic service initialization: ServiceAdapter(descriptor=ServiceDescriptor(interface='com.github.copilot.lang.agent.CopilotAgentProcessService'...
```

**Comment by chzhyu** [comment]
Run into the same problem.
Trun off proxy settings in IDE , use Proxifier instead.  It works, but i don't known why.

**Comment by achyuthe** [comment]
even after downgrading the version. It is forcing me to update the version
<img width="805" alt="image" src="https://user-images.githubusercontent.com/19716671/169603493-9820de20-21db-4395-9c44-c33f176337c2.png">

**Comment by matquant14** [comment]
For what it's worth, nothing in my environment changed when updating the plugin.  I am not using any proxy.  I tried to update to the newest release, 1.1.23.1570, hoping that it might be corrected, but still doesn't work.  Have to downgrade back to 1.1.20.417

**Reply by matquant14** [reply]
Same Issue w/ 1.1.24.1610.  Have to downgrade to 1.1.20.417

![image](https://user-images.githubusercontent.com/43152300/171490020-d1ff5bd8-c9bd-49bc-a5d7-c07c54acc312.png)

**Comment by matquant14** [comment]
Now 1.1.20.417 is incompatible, rendering this plugin completely unusable.

![image](https://user-images.githubusercontent.com/43152300/172850808-10715e1a-10a3-4b19-9fa9-0711ac0952ea.png)

**Reply by matquant14** [reply]
When can we expect a version that works?  I've tried updating to the newest release and still does not work

**Reply by matquant14** [reply]
This solution fixed my issue https://github.com/github-community/community/discussions/18132

Looks there's a problem w/ the CopilotAgent.class file in the core JAR file
